### PR TITLE
Extend contract with shub-image-info

### PR DESCRIPTION
**Pull request by vshlapakov**
Describe extending custom images contract with a new ``shub-images-info`` command.

The PR is complete, awaiting for next shub release and update examples in custom-images-examples repo.

### CODE REVIEW

File: docs/custom-images-contract.rst [hunk 1]
```diff
@@ -24,15 +24,33 @@ Contract statements
 
     docker run myscrapyimage start-crawl
 
-2. Docker image should be able to return a spiders list via ``list-spiders`` command without arguments.
-   ``list-spiders`` should be :ref:`executable and located on the search path <scripts-example>`.
+2. Docker image should be able to return its metadata via ``shub-image-info`` command without arguments.
+   ``shub-image-info`` should be :ref:`executable and located on the search path <scripts-example>`.
```
Comment (chekunkov) [inline docs/custom-images-contract.rst]: We will need to update examples with this new script before merging this then.

File: docs/custom-images-contract.rst [hunk 2]
```diff
@@ -24,15 +24,33 @@ Contract statements
...
+   For now only ``project_type`` and ``spiders`` fields are supported.
 
 .. code-block:: bash
 
-   docker run myscrapyimage list-spiders
+   docker run myscrapyimage shub-image-info
+
+    {
+        "project_type": "casperjs",
```
Comment (vshlapakov) [inline docs/custom-images-contract.rst]: It should be clear which types we're going to support with the protocol. 
I know it's `scrapy` and `casperjs` for sure, any others to include to the doc?

File: docs/custom-images-contract.rst (hunk 1 above)
Comment (vshlapakov) [inline docs/custom-images-contract.rst]: Prepared changes for CasperJS example https://github.com/scrapinghub/custom-images-examples/pull/9/files

File: docs/custom-images-contract.rst (hunk 2 above)
Comment (vshlapakov) [inline docs/custom-images-contract.rst]: Agreed to add a line about available types and validate it server-side.

**Pull request merged**